### PR TITLE
fix(docs): build Hydroflow for playground demo with only WASM features

### PR DESCRIPTION
**Pull request by shadaj**
fix(docs): build Hydroflow for playground demo with only WASM features

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -23,6 +23,19 @@ $ cd ../website_playground
...
+If you're building on macOS, you may need to install the `llvm` package with Homebrew (because the default toolchain has WASM support missing):
+
+```bash
+$ brew install llvm
+```
+
+Then, you'll need to set `TARGET_CC` and `TARGET_AR` environment variables when building the playground:
+
+```bash
+$ TARGET_CC=/usr/local/opt/llvm/bin/clang TARGET_AR=/usr/local/opt/llvm/bin/llvm-ar CARGO_CFG_HYDROFLOW_GENERATE_DOCS="1" RUSTFLAGS="--cfg procmacro2_semver_exempt --cfg super_unstable" wasm-pack build
```
Comment (jhellerstein) [inline docs/README.md:36]: should be 
```
TARGET_CC=$(brew --prefix)/opt/llvm/bin/clang TARGET_AR=$(brew --prefix)/opt/llvm/bin/llvm-ar CARGO_CFG_HYDROFLOW_GENERATE_DOCS="1" RUSTFLAGS="--cfg procmacro2_semver_exempt --cfg super_unstable" wasm-pack build
```

**Review by jhellerstein — COMMENTED** [review]
Please fix brew directory prefix as indicated.

File: docs/README.md (hunk 1 above)
Comment (MingweiSamuel) [reply docs/README.md:36]: quote `"$(brew --prefix)/opt/llvm/bin/clang"`

Comment (MingweiSamuel) [reply docs/README.md:36]: quotes around `"$(brew --prefix)/opt/llvm/bin/clang"` `"$(brew --prefix)/opt/llvm/bin/llvm-ar"`

**Review by MingweiSamuel — CHANGES_REQUESTED** [review]

**Review by jhellerstein — APPROVED** [review]
Please fix brew directory prefix as indicated.

**Review by MingweiSamuel — APPROVED** [review]

**Pull request merged**